server/app/modules/SecurityModule.java

```diff
@@ -77,8 +77,12 @@ protected void configure() {
 
     // you can logout by hitting the logout endpoint, you'll be redirected to root page.
     LogoutController logoutController = new LogoutController();
-    logoutController.setDefaultUrl(routes.HomeController.index().url());
+    logoutController.setDefaultUrl(baseUrl + routes.HomeController.index().url());
+    logoutController.setLocalLogout(true);
     logoutController.setDestroySession(true);
+
+    Boolean logoutProvider = configuration.getBoolean("auth.oidc_provider_logout");
+    logoutController.setCentralLogout(logoutProvider);
```

Comment:
nit: the naming of this variable is confusing since it's a bool but the name calls it a provider?